jboss-eap-clustered-rhel7/README.md

```diff
@@ -6,13 +6,13 @@
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-`Tags: JBoss, EAP, Red Hat,EAP7, CLUSTER`
+`Tags: JBoss, EAP, Red Hat,EAP7, CLUSTER, Azure, Azure VM, JavaEE, RHEL 7
```

Comment:
Since Theresa requested to remove the term Cluster from the Title, removed it from Tags and the folder name as well. Please suggest. Currently, the Tags for the template are "JBoss, Red Hat, EAP 7.2, Azure, Azure VM, JavaEE, RHEL 7.7"